api/graphql/query/CalendarEventInvite/availability.js

```diff
@@ -1,14 +1,13 @@
-import User from 'Services/User';
 import Models from 'Models';
 
-const availability = async (parent) => {
+const availability = async (parent, ags, { loaders }) => {
```

Comment:
ags -> _args. 

Mentioning this the 4th time.

Reply:
I am sorry.
 I fixed it